packages_rs/nextclade/src/io/nextclade_csv.rs

```diff
@@ -252,6 +258,9 @@ impl<W: VecWriter> NextcladeResultsCsvWriter<W> {
     self.add_entry("alignmentScore", &alignment_score)?;
     self.add_entry("alignmentStart", &alignment_start.to_string())?;
     self.add_entry("alignmentEnd", &alignment_end.to_string())?;
+    self.add_entry("refLength", ref_length)?;
+    self.add_entry("totalCoveredNucs", total_covered_nucs)?;
+    self.add_entry("coverage", coverage)?;
```

Comment:
Why? That's just random. No other number in this output is formatted. Also, why 5 and not 3 or 6? 

This will only affect CSV and TSV and other outputs will be different. Makes no sense to me.

I think users should get the whole information and decide how many digits they want to take on their own.